taskcluster/xpi_taskgraph/parameters.py

```diff
@@ -15,6 +15,7 @@
     Optional("shipping_phase"): Any("build", "promote", "ship", None),
     Optional("xpi_name"): Any(basestring, None),
     Optional("xpi_revision"): Any(basestring, None),
+    Optional("additional_shipit_emails"): Any([str], None),
```

Comment:
Should line 13 be removed? I didn't see a docs file in the taskcluster directory.

Reply:
Yes, probably. Or cut down to "Please keep this list sorted", and we could move this line to the top.